built your (mjc) master branch this morning, and got the link error.
Changed code to look like:
```
#if defined(_WIN32)
    struct  _exception *pException
#else
    struct  exception *pException
#endif
```
and the error went away. Hopefully that is fine for everything else --

However, batchvision still doesn't link. Regrettably, I don't have time to chase the following down, as I leave for vacation this afternoon--
```
Undefined                       first referenced
 symbol                             in file
V::VString&Vxa::VSet::getDescription_(V::VString&)const ../lib/libVxa.so
V::VString&Vxa::VType::getDescription_(V::VString&)const ../lib/libVxa.so
ld: fatal: symbol referencing errors. No output written to batchvision
gmake: *** [batchvision] Error 1
```

--- a/software/src/master/src/backend/PFnumeric.cpp
+++ b/software/src/master/src/backend/PFnumeric.cpp
@@ -164,7 +164,6 @@ extern "C" {
  ******************************************/
 
 #if defined(__linux__) || defined(__APPLE__) || (defined (__HP_aCC) && __cplusplus >= 199707L)
-#define USING_FAKE_MATHERR
 struct exception {
     int type;
     char *name;
@@ -189,10 +188,8 @@ struct exception {
 int __cdecl matherr (
 #if defined(_WIN32)
     struct  _exception *pException
-#elif defined(USING_FAKE_MATHERR)
-    struct   exception *pException
 #else
-    struct __exception *pException
+    struct   exception *pException
 #endif
 ) {
     static const double iNaN (NaNQ);

